charts/seed-monitoring/charts/prometheus/rules-tests/kube-etcd3.rules.test.yaml

```diff
@@ -77,6 +77,6 @@ tests:
         pod: etcd
         job: kube-etcd3
       exp_annotations:
-        description: Etcd3 pod etcd has seen 7 proposal failures
+        description: Etcd3 pod etcd has seen 85.5 proposal failures
```

Comment:
Please change it to `81` as mentioned above.

Reply:
Done